content/contribute/_index.md

```diff
@@ -32,26 +32,51 @@ Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams
 
 Now you can build and run BotKube by one of the following ways
 
-### Build the container image
+### Build and install on Kubernetes
+
+1. Build BotKube and create a new container image tagged as `ghcr.io/kubeshop/botkube:v9.99.9-dev`. Choose one option:
+
+    - **Single target build for your local K8s cluster**
+
+      This is ideal for running BotKube on a local cluster, e.g. using [kind](https://kind.sigs.k8s.io) or [`minikube`](https://minikube.sigs.k8s.io/docs/).
+
+      Remember to set the `IMAGE_PLATFORM` env var to your target architecture. For example, the command below builds the `linux/arm64` target. By default, the build targets `linux/amd64`.
+
+         ```sh
+         IMAGE_PLATFORM=linux/arm64 make container-image-single
+         docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev <your_account>/botkube:v9.99.9-dev
+         docker push <your_account>/botkube:v9.99.9-dev
+         ```
+      Where `<your_account>` is Docker hub account to which you can push the image.
+
+    - **Multi-arch target builds for any K8s cluster**
+
+      This is ideal for running BotKube on remote clusters.
+
+      When tagging your dev image take care to add your target image architecture as a suffix. For example, in the command below we added `-amd64` as our target architecture.
+
+      This ensures the image will run correctly on the target K8s cluster.
+
+      > **Note**
+      > This command takes some time to run as it builds the images for multiple architectures.
 
-1. This will build BotKube and create a new container image tagged as `ghcr.io/kubeshop/botkube:v9.99.9-dev`
    ```sh
-   make build
    make container-image
    docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
    docker push <your_account>/botkube:v9.99.9-dev
    ```

```

Comment:
please add a padding to align with a note (I also fixed that on the `botkube` repo: https://github.com/kubeshop/botkube/pull/665)

Reply:
I'll have a look. I think GoLand is removing the padding.

The content was copied directly from [the PR](https://github.com/kubeshop/botkube/pull/665)